plugins/scaffolder-backend-module-confluence-to-markdown/src/actions/confluence/confluenceToMarkdown.examples.test.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { PassThrough } from 'stream';
+import { createConfluenceToMarkdownAction } from './confluenceToMarkdown';
+import { getVoidLogger } from '@backstage/backend-common';
+import { UrlReader } from '@backstage/backend-common';
+import { ConfigReader } from '@backstage/config';
+import { ScmIntegrations } from '@backstage/integration';
+import { setupRequestMockHandlers } from '@backstage/backend-test-utils';
+import mockFs from 'mock-fs';
+import os from 'os';
+import { readFile, writeFile, createWriteStream } from 'fs-extra';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { examples } from './confluenceToMarkdown.examples';
+import yaml from 'yaml';
+
+jest.mock('fs-extra', () => ({
```

Comment:
You probably do not want to combine mock-fs with mocking fs-extra, I'd try to stick to just mock-fs to keep things simple. Was there anything particular that sent you on this route?